pkg/utils/openshift/utils_test.go

```diff
@@ -39,3 +42,69 @@ func TestOpenShiftVersion_MapKnownVersion(t *testing.T) {
 		assert.Equal(t, v.expectedOCPVersion, MapKnownVersion(v.info).Version, v.label+": expected OCP version to match")
 	}
 }
+
+func TestGetConsoleYAMLSample(t *testing.T) {
+	var inputYaml = `
+apiVersion: app.kiegroup.org/v2
+kind: KieApp
```

Comment:
If you're parsing a DeploymentConfig, also provide a DC here.

Reply:
convert to DC. 
and I think we need two other annotations for TargetResource which contains API version and kind. Please check the code.